### PR TITLE
plugins.rtvs: fixed livestream

### PR DESCRIPTION
**Pull request by back-to**
before

```
$ streamlink https://www.rtvs.sk/televizia/live-1
[cli][info] Found matching plugin rtvs for URL https://www.rtvs.sk/televizia/live-1
error: No playable streams found on this URL: https://www.rtvs.sk/televizia/live-1
```

after

```
$ streamlink https://www.rtvs.sk/televizia/live-1
[cli][info] Found matching plugin rtvs for URL https://www.rtvs.sk/televizia/live-1
[cli][info] Available streams: 360p (worst), 480p, 720p, 1080p (best)
[cli][info] Opening stream: 1080p (hls)
```

### CODE REVIEW

File: src/streamlink/plugins/rtvs.py [hunk 1]
```diff
@@ -2,65 +2,46 @@
...
+        videos = parse_json(res.text, schema=validate.Schema({
+            "clip": {
+                "sources": [{
+                    "src": validate.url(),
+                    "type": str,
+                }],
+            }},
+            validate.get(("clip", "sources")),
+            validate.filter(lambda n: n["type"] == "application/x-mpegurl"),
+        ))
```
Comment (bastimeyer) [inline src/streamlink/plugins/rtvs.py:42]: Just a bit of nit-picking, but if you replace `"type": str` with `"type": "application/x-mpegurl"`, then you don't need `validate.filter` (unless you don't want to raise an error if no list item matches the dict schema).

**Review by bastimeyer — APPROVED** [review]
Plugin is looking good, but I'm not going to merge right away in case you want to change the validation schema (which is not necessary though).

File: src/streamlink/plugins/rtvs.py (hunk 1 above)
Comment (back-to) [reply src/streamlink/plugins/rtvs.py:42]: @bastimeyer 
I don't want the error `Unable to validate JSON:`, if there are other `types` than `application/x-mpegurl`

Comment (bastimeyer) [reply src/streamlink/plugins/rtvs.py:42]: Yes, you are right, I misread `validate_sequence`
https://github.com/streamlink/streamlink/blob/5d1227e5977d3c1e98459799769ac6b207aab3dc/src/streamlink/plugin/api/validate.py#L352

**Pull request merged**